Firestore/core/test/firebase/firestore/testutil/testutil.h

```diff
@@ -40,6 +46,24 @@ inline model::ResourcePath Resource(absl::string_view field) {
   return model::ResourcePath::FromString(field);
 }
 
+// Version is epoch time in microseconds. This helper is defined so to match
+// SDKs in other platform.
+inline model::SnapshotVersion Version(int64_t version) {
+  return model::SnapshotVersion{
+      Timestamp{/*seconds=*/version / 1000,
```

Comment:
If time is in microseconds, should this be `version / (1000 * 1000)`?

Also, optionally consider something like:
```cpp
namespace chr = std::chrono;
auto timepoint = chr::time_point<chr::system_clock>(chr::microseconds(version));
return model::SnapshotVersion{Timestamp::FromTimePoint(timepoint)};
```

This is a little more verbose, but arguably more readable and saves doing arithmetic operations manually.